tests/loss/test_DINOLoss.py

```diff
@@ -0,0 +1,269 @@
+import itertools
+import unittest
+
+import numpy as np
+import torch
+from torch import nn
+import torch.nn.functional as F
+
+from lightly.loss import dino_loss
+from lightly.loss import DINOLoss
+
+
+class FacebookDINOLoss(nn.Module):
```

Comment:
Would drop the `Facebook` part here. Or are there various implementations of the same loss?